examples/integration-tutorials/using-the-gmail-connector/gmail_connector.bal

```diff
@@ -0,0 +1,103 @@
+import ballerina/io;
+import ballerina/http;
+import ballerina/log;
+import wso2/gmail;
+
+gmail:GmailConfiguration gmailConfig = {
```

Comment:
We need to add method and variable comments

Reply:
Done.